src/scripts/code-copy.js

```diff
@@ -0,0 +1,45 @@
+document.addEventListener('DOMContentLoaded', function () {
+  const code = document.querySelectorAll('pre[class*="language"]');
+  const url = window.location.href;
+  let lang = 'en';
+
+  if (url.includes('/fr/')) {
+    lang = 'fr';
+  }
+
+  const buttonClasses = [
+    'd-block',
+    'mt-200',
+    'code-copy-button'
+  ]
+
+  code.forEach((pre) => {
+    const button = document.createElement('gcds-button');
+    button.classList.add(...buttonClasses);
+    button.setAttribute('button-role', 'secondary');
+    button.setAttribute('size', 'small');
+
+    if (lang === 'en') {
+      button.innerHTML = 'Copy';
+    } else {
+      button.innerHTML = 'Copie';
+    }
```

Comment:
Can we simplify this? Here's an example:

```javascript
    // Define this or if you want, grab it from a language file aka i18n
    const copyText = {
      'en' : 'Copy',
      'fr' : 'Copié'
    }
```

and use it accordingly
```suggestion
    button.innerHTML = copyText[lang]
    }
```

It will reduce our errors and we only need to change it in one place. I already see some say `Copie` and others say `Copié`